Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml

```diff
@@ -31,10 +31,10 @@
   id: FoodBreadBunBottom
   parent: FoodBreadSliceBase
   name: bottom bun
-  description: It's time to start building the burger tower. 
```

Comment:
If possible, try to avoid touching random parts of the file without a good reason! This is probably your IDE doing it automatically :(

I usually do the edit in notepad if its easy enough!

Reply:
ah sorry, yeah i'm assuming one of my extensions did it